bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/Bpmn.java

```diff
@@ -353,7 +357,7 @@ public static BpmnModelInstance createEmptyModel() {
   public static ProcessBuilder createProcess() {
     final BpmnModelInstance modelInstance = INSTANCE.doCreateEmptyModel();
     final Definitions definitions = modelInstance.newInstance(Definitions.class);
-    definitions.setTargetNamespace(BPMN20_NS);
+    setModelDefinition(definitions);
```

Comment:
❌ The name of this method isn't true. It doesn't set the model definition. It sets some properties on the definitions object we already created.

I'd like it better if you'd create a method `createModelDefinition` which would then include creating the definitions (`modelInstance.newInstance(Definitions.class);`) and returning it to set it on the `modelInstance`.

Reply:
I like this solution 👍